recipe/meta.yaml

```
@@ -1,44 +1,51 @@
 {% set name = "phconvert" %}
-{% set version = "0.9" %}
-{% set sha256 = "7049ec6b57e7571988eabf7f1a3025b129c9000e873374fbb85c2d837d8a8d17" %}
+{% set version = "0.9.1" %}
+{% set sha256 = "ec2be3a9cad9674decae1749c9ab1441794bbff289a2ce4213856b04786853ea" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/Photon-HDF5/{{ name }}/archive/{{ version }}.tar.gz
+  url: https://github.com/Photon-HDF5/phconvert/archive/{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
   noarch: python
-  number: 2
-  script: python -m pip install -vv --no-deps --ignore-installed .
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} 
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  number: 0
 
 requirements:
   host:
-    - pip
     - python
-    - setuptools
-
+    - setuptools >=64
+    - setuptools-scm >=6.0
+    - pip
   run:
-    - python >=3.5
-    - setuptools
-    - numpy >=1.9
+    - python >=3.6
+    - numpy >=1.19
     - pytables
+    - importlib_resources  # [py<=310]
```

```suggestion
    - importlib_resources
```

just remove this "if statement". Conda doesn't support them in the same way that pip does.

You can have it if you want to have a huge build matrix and build out separate packages for python 3.8, 3.9. 3.10, 3.11, 3.12, and pypy, but it really isn't critical that it "not be installed" with python 3.8-3.10.

So simplify your recipe, and force users to install importlib_resorces with your package.

---

Otherwise, with conda-forge, you would have to remove the line that says `noarch: python`.

---

After double checking that I can still install the` importlib_resources` back-port of `importlib.resrouces` on the newer version of Python, I think that yes, forcing install is the best option.

Thank you for your help.